code/config.ss

```diff
@@ -78,63 +76,63 @@
       (topics optimization dynamic-programming search))
 
      ;;; Other problems
-     
+
      ((slug . bob)
       (uuid . "4cf6697c-ad9e-426c-a2d8-6c3695812421")
       (core . #f)
       (unlocked-by)
       (difficulty . 1)
       (topics string))
-     
+
      ((slug . raindrops)
       (uuid . "9f13b913-a650-4cbf-a392-4b5614e1aa2a")
       (core . #f)
       (unlocked-by)
       (difficulty . 1)
       (topics))
-     
+
      ((slug . robot-name)
       (uuid . "62a5a71c-a9eb-416b-809c-c862a4828e8b")
       (core . #f)
       (unlocked-by)
       (difficulty . 1)
       (topics))
-     
+
      ((slug . phone-number)
       (uuid . "8ed74044-2361-459e-aca5-abd73ed5c1cb")
       (core . #f)
       (unlocked-by)
       (difficulty . 1)
       (topics string))
-     
+
      ((slug . nucleotide-count)
       (uuid . "d72cf2c3-9f30-4df1-a69b-04c638bb2426")
       (core . #f)
       (unlocked-by rna-transcription)
       (difficulty . 3)
       (topics string hashtable))
-     
+
      ((slug . list-ops)
       (uuid . "903374d2-8155-4fe3-bf90-5b6359c7b5cc")
       (core . #f)
       (unlocked-by)
       (difficulty . 1)
       (topics))
-     
+
      ((slug . scrabble-score)
       (uuid . "5b44ccd4-b1f4-4b86-89c8-f1dec5905ccb")
       (core . #f)
       (unlocked-by rna-transcription)
       (difficulty . 3)
       (topics string list))
-     
+
      ((slug . word-count)
       (uuid . "9460b65d-dc80-4a95-8782-b395d2cc979e")
       (core . #f)
       (unlocked-by nucleotide-count)
```

Comment:
Travis is breaking because of this line.

It is a good reminder that all changes to `config.ss` should have `config.json` regenerated, so that `configlet` can check for incompatibilities.

What do you think is a good core exercise to use for unlocking `word-count`? It must be core.

Reply:
I think that `anagram` would be a good choice for unlocking `word-count` given the current list of core exercises.

I will go ahead and change that so this is passing.  let me know what you think about it.

Reply:
Made the change

Reply:
I always make before pushing

Reply:
I'm not sure why the exercise dependencies weren't being added to the `config.json` then.

Oh well. What's your take on this PR now?

I think it is ready (even if it isn't perfect).

It will probably be easier to add future improvements once this is no longer pending.

Reply:
The biggest requirements I guess are no conflicts and the output matches `confliglet`'s formatting

Reply:
Ah they weren't being added because of scheme-case. Your pull will change that. (Currently setting `unlocked-by` to `unlocked_by` works, too)

Reply:
Cool. So are you ready for this to get merged?